ddtrace/tracer/option.go

```diff
@@ -484,3 +485,13 @@ func StackFrames(n, skip uint) FinishOption {
 		cfg.SkipStackFrames = skip
 	}
 }
+
+// boolEnv returns the parsed boolean value of an environment variable, or
+// d if it fails to parse.
+func boolEnv(key string, d bool) bool {
```

Comment:
I think `d` should be `default` for documentation purposes.

Reply:
`default` is a Go [keyword](https://golang.org/ref/spec#Keywords). We can do `def` or 'otherwise' or something else if you prefer.